bigflow/__init__.py

```diff
@@ -2,6 +2,8 @@
 from .workflow import Workflow, Definition
 from .configuration import Config
 
+from bigflow._version import __version__
```

Comment:
Do we need a separate module to hold the version?

Reply:
Imo it's clear where you can find current version

Reply:
A separate file. It is 'exec'ed from setup.py directly (so bigflow/__init__.py is not involved).
We can't put version to '__init__' directly, because otherwise 'setup.py' will try to import 'bigflow' before it is installed (or we need to duplicate version in 2 places).

Reply:
true